contrib/externs/angular-1.5.js

```diff
@@ -1795,6 +1795,20 @@ angular.$locale = function() {};
  */
 angular.$locale.prototype.id;
 
+/**
+ * @typedef {{
+ *     CURRENCY_SYM: (string),
+ *     GROUP_SEP: (string),
+ *     DECIMAL_SEP: (string)
+ * }}
+ */
+angular.$locale.NUMBER_FORMATS;
```

Comment:
It I put `@type` I get this error (in my project):
```
ERR! compile ...: ERROR - Bad type annotation. Unknown type angular.NUMBER_FORMATS
ERR! compile  * @type {angular.NUMBER_FORMATS}
ERR! compile           ^
ERR! compile 
ERR! compile 
ERR! compile 1
ERR! compile  error(s), 
ERR! compile 0
ERR! compile  warning(s)
ERR! compile 
ERR! Process exited with non-zero status, see log for more detail: 1 
```
:-(
When I see the rest of the file I think that the @typedef is the best...